### PR TITLE
Feature#18/crud posts

### PR DESCRIPTION
**Pull request by juperameza**
# Description
 - What?: I've added the crud for the posts using scaffold and simple form
 - Why?: To have the crud ready to use just needing to implement the styles

  For mor information see issue #18 
  ## Screenshots 
image
image
image

### CODE REVIEW

File: app/helpers/posts_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module PostsHelper
+end
```
Comment (Frcomparan) [inline app/helpers/posts_helper.rb:2]: If you don't use it, you should remove it.

**Review by Frcomparan — COMMENTED** [review]
Se ve bien, solo un comentario.

File: app/views/posts/_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div id="<%= dom_id post %>">
+    <%= post.title%>
+    <%= post.description%>
+    <%= post.picture%>
+    <%= post.price%>
+    <%= post.type_id ? "Producto": "Servicio" %>
+</div>
```
Comment (YilKanda) [inline app/views/posts/_post.html.erb:7]: Check your indentation (Space: 2)

**Review by YilKanda — APPROVED** [review]

**Review by Frcomparan — APPROVED** [review]

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**